docs/cse/rules/write-first-seen-rule.md

```diff
@@ -28,13 +28,13 @@ First Seen rules allow you to generate a Signal when behavior by an Entity (such
 * High severity EDR alert seen for the first time
 * MFA acceptance from first seen device
 
-A First Seen rule is different from other CSE rule types in that you don’t define the criteria for firing a Signal. Instead, the rule expression in a First Seen rule is simply a filter condition that defines what incoming Records the rule will apply to. For each First Seen rule, CSE automatically creates a baseline model of normal behavior evidenced by Records that match the Rule Expression. After the baseline period is completed, when an incoming Record includes matching activity not seen during the baseline period, the rule creates a Signal.
+A First Seen rule is different from other CSE rule types in that you don’t define the criteria for firing a Signal. Instead, the rule expression in a First Seen rule is simply a filter condition that defines what incoming Records the rule will apply to. For each First Seen rule, CSE automatically creates a baseline model of normal behavior evidenced by Records that match the Rule Expression. After the baseline learning period is completed, when an incoming Record includes matching activity not seen during the baseline learning period, the rule creates a Signal.
 
 For example, for the “First time a user logged in from a new geographic location” use case, CSE will build a baseline model of all the geolocations from where a logon event is seen for the Entity (user). Once the baselining period is complete, CSE will create a Signal for every new geolocation detected and incrementally add to the baseline.
 
 ## Example rule
 The screenshot below shows a First Seen rule in the CSE rules editor. For an explanation of the configuration options, see [Configure a First Seen Rule](#configure-a-first-seen-rule), below.
-<img src={useBaseUrl('img/cse/first-seen-rule.jpg')} alt="Example First Seen Rule Definition"/>
+<img src={useBaseUrl('img/cse/first-seen-rule-example-new.jpg')} alt="Example First Seen Rule Definition"/>
```

Comment:
Rather than creating a new image w/new file name here, best practice is to override existing image with your new image. We don't want to retain old unused images as they add to build times.